notifications/src/main/java/org/openedx/notifications/presentation/NotificationsAnalytics.kt

```diff
@@ -6,26 +6,96 @@ interface NotificationsAnalytics {
 }
 
 enum class NotificationsAnalyticsEvent(val eventName: String, val biValue: String) {
-    DISCUSSION_PERMISSION_TOGGLE(
-        eventName = "Notification:Discussion Permission Toggle",
-        biValue = "edx.bi.app.notification.discussion.permission.toggle"
+    PUSH_NOTIFICATIONS_SETTINGS(
+        eventName = "Notification:Push Notifications Settings",
+        biValue = "edx.bi.app.notification.push_notifications_settings"
+    ),
+    DISCUSSION_PREFERENCE_TOGGLE(
+        eventName = "Notification:Discussion Preference Toggle",
+        biValue = "edx.bi.app.notification.preference.discussion.toggle"
+    ),
+    PUSH_PREFERENCES_BATCH_TOGGLE_STATE(
+        eventName = "Notification:Preferences Toggle Batch State",
+        biValue = "edx.bi.app.notification.preference.batch.toggle"
+    ),
+    SYSTEM_PERMISSION_DIALOG_VIEWED(
+        eventName = "Notification:System Permission Dialog Viewed",
+        biValue = "edx.bi.app.notification.system.permission.dialog.viewed"
+    ),
+    SYSTEM_PERMISSION_DIALOG_ACTION(
+        eventName = "Notification:System Permission Dialog Action",
+        biValue = "edx.bi.app.notification.system.permission.dialog.action"
+    ),
+    APP_PERMISSION_RATIONALE_DIALOG_VIEWED(
+        eventName = "Notification:App Permission Rationale Dialog Viewed",
+        biValue = "edx.bi.app.notification.app.permission.dialog.viewed"
+    ),
+    APP_PERMISSION_RATIONALE_DIALOG_ACTION(
+        eventName = "Notification:App Permission Rationale Dialog Action",
+        biValue = "edx.bi.app.notification.app.permission.dialog.action"
+    ),
```

Comment:
there are unused.

Reply:
I left it here on purpose so that in the upcoming PR, we still have the correct analytics and keys.